streams/actions/src/main/java/com/hortonworks/streamline/streams/actions/topology/service/TopologyActionsService.java

```diff
@@ -355,4 +375,12 @@ public StreamCatalogService getCatalogService() {
         return catalogService;
     }
 
+    private Exception convertThrowableToException(Throwable e) {
+        if (e.getCause() instanceof Exception) {
+            return (Exception) e.getCause();
```

Comment:
Findbugs warning.
 
```
[INFO] Unchecked/unconfirmed cast from Throwable to Exception of return value in 
com.hortonworks.streamline.streams.actions.topology.service.TopologyActionsService.convertThrowableToExce
ption(Throwable) [com.hortonworks.streamline.streams.actions.topology.service.TopologyActionsService] At 
TopologyActionsService.java:[line 380] BC_UNCONFIRMED_CAST_OF_RETURN_VALUE
```

You could assign the `cause` to a local variable and use that in the condition and cast to avoid this.

Reply:
Sadly I missed this comment and just fixed it with same approach instead of annotation. Thanks for guiding.